### PR TITLE
Remove seatedSectionUsers subcollection on auditorium section deletion

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1733

### CODE REVIEW

File: functions/venue.js [hunk 1]
```diff
@@ -300,6 +300,26 @@ exports.setAuditoriumSections = functions.https.onCall(
...
+        .doc(sectionId)
+        .collection("seatedSectionUsers")
+        .listDocuments()
+        .then((array) => {
+          array.map((userDoc) => batchForSections.delete(userDoc));
+          return batchForSections.commit();
+        })
+        .catch((e) => {
+          throw e;
+        });
```
Comment (goran-peoski-work) [inline functions/venue.js:320]: This is functionally equivalent with:
```suggestion
```

File: functions/venue.js [hunk 2]
```diff
@@ -300,6 +300,26 @@ exports.setAuditoriumSections = functions.https.onCall(
...
+      const sectionId = sections[currentNumberOfSections - i].id;
+      admin
+        .firestore()
+        .collection("venues")
+        .doc(venueId)
+        .collection("sections")
+        .doc(sectionId)
+        .collection("seatedSectionUsers")
+        .listDocuments()
+        .then((array) => {
```
Comment (goran-peoski-work) [inline functions/venue.js:314]: How will this entire loop act if a section with 200 seated users is being deleted?
Will it hit a limit of time or memory?
Will an error in this prevent adding or deleting sections?

**Review by goran-peoski-work — COMMENTED** [review]
The logic is there. Not sure if the behavior is safe enough not to cause regressions in other areas

**Comment by sunny-viktoryia** [comment]
not going to merge the PR, auditoriums don't have sections now.

**Pull request closed without merging**